hack/update-mocks.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
please add `verify-mocks.sh`, add add these files to `update-all.sh` and `verify-all.sh`.